online_pmi

```diff
@@ -2,21 +2,23 @@
 
 """Align words based on stepwise EM alignments with PMI scores."""
 
-from collections import defaultdict
+from collections import defaultdict, Counter
 import itertools as it
 import sys
 import igraph, utils
```

Comment:
I agree. Unfortunately, these import issues prevent the system from working on windows (at least on my version).